brx/gulpfiles/lib/erb_helper.rb

```diff
@@ -0,0 +1,40 @@
+require "pathname"
```

Comment:
Don't we need a similar file for the testing? Maybe we can put it in `lib/web_console` and use as an library instead.